Source/Directory.Build.props

```diff
@@ -2,7 +2,7 @@
 
   <!-- Target framework and package configuration -->
   <PropertyGroup>
-    <Version>3.0.12</Version>
+    <Version>3.1.12</Version>
```

Comment:
I imagine you're bumping to 3.1 because of the API change? If so, though, wouldn't 3.1.0 make more sense?

Reply:
It would. Updated